rollup.config.dev.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const path = require('path');
+const resolve = require('rollup-plugin-node-resolve');
+
+const addPath = files => files.map(f => path.join(__dirname, f));
+
+// Tag well-known warnings so we can silence them.
+const IGNORE_WARNINGS = {
+  // Some third party code emits warnings from parsing their
+  // UMD/commonjs exporting.
+  THIS_IS_UNDEFINED: addPath(['node_modules/marked/lib/marked.js']),
+  // Some third party libraries may have optional modules.
+  MISSING_EXPORT: addPath(['lib/third_party/three/GLTFLoader.js']),
+};
+
+const onwarn = (warning, warn) => {
+  const filesToIgnore = IGNORE_WARNINGS[warning.code];
+  if (filesToIgnore && filesToIgnore.indexOf(warning.id) !== -1) {
+    return;
+  }
+
+  warn(warning);
+};
```

Comment:
I recognize this code could be put in it's module. I can do this in this PR if you all want.